### PR TITLE
dev

### PR DESCRIPTION
**Pull request by matthdsm**
- Support for bamprocessing with ELPREP
- Improved sample grouping
- Meta value clean up
- bump modules
- fix multiqc report to include all results
- fix duplicate marking with bamsormadup
- drop post alignment sorting, doesn't make sense as we do sort/markdup afterwards
- Add option to use elprep for alignment postprocessing

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-preprocessing/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the CenterForMedicalGeneticsGhent/nf-cmgg-preprocessing _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -237,7 +238,7 @@ manifest {
     defaultBranch   = "main"
     mainScript      = "main.nf"
     nextflowVersion = "!>=22.10.0"
-    version         = "1.1.1"
+    version         = "1.1.2dev"
```
Comment (nvnieuwk) [inline nextflow.config:241]: This isn't dev anymore after merge ;)

**Review by nvnieuwk — APPROVED** [review]
LGTM!

File: nextflow.config (hunk 1 above)
Comment (matthdsm) [reply nextflow.config:241]: good catch!

Comment (matthdsm) [inline nextflow.config:241]:
```suggestion
    version         = "1.1.2"
```

**Pull request merged**